packages/safe-tools-client/app/components/future-payments-list/index.gts

```diff
@@ -1,49 +1,110 @@
+    let nextMonth = addMonths(now, 1);
+    let firstDateNextMonth = new Date(nextMonth.getFullYear(), nextMonth.getMonth(), 1, 0, 0, 0, 0);
+    let lastDateNextMonth = new Date(nextMonth.getFullYear(), nextMonth.getMonth(), lastDayOfMonth(nextMonth).getDate(), 23, 59, 0, 0);
+    return this.futurePaymentsResource.value.filter((s: ScheduledPayment) => s.payAt >= firstDateNextMonth && s.payAt <= lastDateNextMonth);
+  }
+
+  get nextFewMonthsPayments(): ScheduledPayment[] {
+    if (!this.futurePaymentsResource.value) return [];
+    let now = new Date();
+    let nextMonth = addMonths(now, 1);
+    let lastDateNextMonth = new Date(nextMonth.getFullYear(), nextMonth.getMonth(), lastDayOfMonth(nextMonth).getDate(), 23, 59, 0, 0);
+    return this.futurePaymentsResource.value.filter((s: ScheduledPayment) => s.payAt > lastDateNextMonth);
   }
 
-  @action
-  async addFunds() {
-    //TODO: implement add fund action
+  @task *loadFuturePaymentTask(chainId: number): TaskGenerator<ScheduledPayment[]> {
+    return yield this.scheduledPayments.fetchScheduledPayments(chainId, new Date());
   }
 
+  @use futurePaymentsResource = resource(() => {
+    if (!this.hubAuthentication.isAuthenticated) {
+      return {
+        error: false,
+        isLoading: false,
+        value: [],
+      };
+    }
+
+    const state = new TrackedObject({
+      isLoading: false,
+      value: undefined as ScheduledPayment[] | undefined,
+      error: undefined,
+    });
+
+    let chainId = this.network.chainId;
+
+    (async () => {
+      try {
+        state.value = await taskFor(this.loadFuturePaymentTask).perform(chainId);
+      } catch (error) {
+        console.log(error);
+        state.error = error;
+      } finally {
+        state.isLoading = false;
+      }
+    })();
+
+    return state;
+  });
+
   <template>
-    <BoxelActionContainer
+    <BoxelActionContainer 
       class="future-payments-list"
       as |Section ActionChin|>
-      {{#if (lt this.futurePayments.length 1)}}
-        <Section class="future-payments-list__no-payments-section">
+      {{#if (lt this.futurePaymentsResource.value.length 1)}}
+        <Section class="future-payments-list__no-payments-section" data-test-no-future-payments-list>
           <div class="future-payments-list__no-payments-title">Schedule your first payment</div>
           <div class="future-payments-list__no-payments-description">Your future payments will show up here. This is where you can check on the status of your transactions and view important messages.</div>
         </Section>
       {{else}}
-        <Section @title="Future Payments">
-          <div>TODO: Show payment card components on time brackets here</div>
+        <Section @title="Future Payments" class="future-payments-list__payments-section" data-test-future-payments-list>
+          <div class="future-payments-list__payments-section-time-brackets">
+            <TimeBracket @title={{"next hour"}} @scheduledPayments={{this.nextHour}}/>
```

Comment:
Since `TimeBracket` is tightly coupled with scheduled payments, I think it would make sense to name it `ScheduledPaymentTimeBracket`